uint8 id

float32 Rad_distance 
float32 Hor_angle 
float32 Rad_in_velocity  
float32 Rad_go_velocity  
float32 RCS     
float32 Obj_size  
float32 Y_distance  
float32 X_distance  
float32 r_l_Sub_velocity  
float32 in_Sub_velocity  
float32 l_r_Sub_velocity  
float32 go_Sub_velocity 
